src/PVRFilmonData.cpp

```diff
@@ -43,7 +43,7 @@ PVRFilmonData::~PVRFilmonData(void) {
 }
 
 bool PVRFilmonData::Load(std::string user, std::string pwd) {
-	P8PLATFORM::CLockObject lock(m_mutex);
+	std::unique_lock<std::recursive_mutex> lock(m_mutex);
```

Comment:
Any reason for using this or would the simpler `std::lock_guard<std::mutex>` have worked instead?

I'm just curious!

Reply:
Not direct, only was the `std::recursive_mutex` more equal to the way used by p8-platform.

Reply:
Is updated, the recursive_mutex makes no sense by this calls.